package.json

```diff
@@ -11,8 +11,10 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.53",
     "@types/react-dom": "^16.9.8",
+    "@types/react-router-dom": "^5.1.6",
```

Comment:
Thanks for installing this! Completely forgot that with typescript you have to install both `react-router-dom` and `@types/react-router-dom`